contracts/LifToken.sol

```diff
@@ -105,22 +87,11 @@ contract LifToken is MintableToken, Pausable {
      @return true if the call function was executed successfully
    */
   function transferDataFrom(address from, address to, uint256 value, bytes data) public whenNotPaused returns (bool) {
-
     require(to != address(this));
 
-    uint256 allowance = allowed[from][tx.origin];
-
-    balances[from] = balances[from].sub(value);
-    balances[to] = balances[to].add(value);
-    allowed[from][tx.origin] = allowance.sub(value);
-
-    if (to.call(data)) {
-      Transfer(tx.origin, to, value);
-      return true;
-    } else {
-      return false;
-    }
-
+    super.transferFrom(from, to, value);
+    require(to.call(data));
```

Comment:
I think it should be first the require to save some gas, because if it fails later the gas used by transfer will be consumed.

Reply:
Agreed, just changed it in d818a37